aiohttp/typedefs.py

```diff
@@ -35,7 +35,13 @@
 Byteish = Union[bytes, bytearray, memoryview]
 JSONEncoder = Callable[[Any], str]
 JSONDecoder = Callable[[str], Any]
-LooseHeaders = Union[Mapping[Union[str, istr], str], _CIMultiDict, _CIMultiDictProxy]
+LooseHeaders = Union[
+    Mapping[str, str],
+    Mapping[istr, str],
```

Comment:
This is a nice fix. Might want to backport this one

Reply:
Yeah, I'll backport the non-test changes.